fabric/src/test/java/TestRegex.java

```diff
@@ -885,13 +884,6 @@ public void WarTowerBar_TOWER_BAR_PATTERN() {
         p.shouldMatch("§3[SEPC] §bEfilim South Plains Tower§7 - §4❤ 390000§7 (§610.0%§7) - §c☠ 1300-1949§7 (§b0.5x§7)");
     }
 
-    @Test
-    public void GuildSeasonLeaderboardLabelParser_GUILD_SEASON_LEADERBOARD_PATTERN() {
```

Comment:
I opted to remove the test case as it would just be absurdly long but I can easily add it back if desired